Xamarin.Forms.Platform.iOS/Cells/CellRenderer.cs

```diff
@@ -58,11 +76,15 @@ protected void UpdateBackground(UITableViewCell tableViewCell, Cell cell)
 		{
 			var uiBgColor = UIColor.White; // Must be set to a solid color or blending issues will occur
 
-			var  defaultBgColor = cell.OnThisPlatform().DefaultBackgroundColor();
+			var defaultBgColor = cell.OnThisPlatform().DefaultBackgroundColor();
 			if (defaultBgColor != Color.Default)
 			{
 				uiBgColor = defaultBgColor.ToUIColor();
 			}
+			else if (cell.IsSet(Cell.BackgroundColorProperty))
```

Comment:
It will not be possible to set the background color if the `DefaultBackgroundColor` is equal to `Color.Default`?

Reply:
Extended the first if-statement like this: `if (!cell.IsSet(Cell.BackgroundColorProperty) && defaultBgColor != Color.Default)`